File: python/challenge_5/pages/home_page.py
from pylenium.driver import Pylenium
from pylenium.element import Element


class HomePage:
    def __init__(self, py: Pylenium):
        self.py = py

    _IMG_HOTEL_LOGO = '.hotel-logoUrl'

    def get_hotel_logo(self) -> Element:
        return self.py.get(self._IMG_HOTEL_LOGO)

File: js/challenge_2/test/pageobjects/home.page.js

class HomePage {
    /**
     * define selectors using getter methods
     */
    get imgHotelLogo () { return $('.hotel-logoUrl') }
    get imgRooms () { return $('.hotel-img') }
    get imgMaps () { return $('.map img') }

}

module.exports = new HomePage();
